src/Symfony/Component/ErrorHandler/Resources/views/traces_text.html.php

```diff
@@ -20,15 +20,15 @@
                 <?php if ($exception['trace']) { ?>
                 <pre class="stacktrace">
 <?php
-                    echo $exception['class'].":\n";
+                    echo $this->escape($exception['class']).":\n";
                     if ($exception['message']) {
-                        echo $exception['message']."\n";
+                        echo $this->escape($exception['message'])."\n";
                     }
 
                     foreach ($exception['trace'] as $trace) {
                         echo "\n  ";
                         if ($trace['function']) {
-                            echo 'at '.$trace['class'].$trace['type'].$trace['function'].'('.(isset($trace['args']) ? $this->formatArgsAsText($trace['args']) : '').')';
+                            echo $this->escape('at '.$trace['class'].$trace['type'].$trace['function']).'('.(isset($trace['args']) ? $this->formatArgsAsText($trace['args']) : '').')';
                         }
                         if ($trace['file'] && $trace['line']) {
                             echo($trace['function'] ? "\n     (" : 'at ').strtr(strip_tags($this->formatFile($trace['file'], $trace['line'])), [' at line '.$trace['line'] => '']).':'.$trace['line'].($trace['function'] ? ')' : '');
```

Comment:
here too?

Reply:
let's remove the extra brackets and add a space after the echo anyway :)

Reply:
the args are escaped (call to `strip_tags`), remaining `line` which should not contains any html code

Reply:
@nicolas-grekas: I guess the space between `echo` and `(` keeps being removed (incorrectly) by the CS fixer because of https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4817 (for which I have already proposed two fixes but...)